### DISCUSSION TITLE
Route conflict with `react-router-dom` and` vercel.json`

### DISCUSSION THREAD

**Discussion by EuCarlos** [vercel/vercel#6955] [Help]
I am using vitejs to create react applications and have deployed it to vercel. I created the routes using react-router-dom and for the routes to work well in production I created the following script:
**vercel.json**
```
{
    "rewrites":  [
        { 
            "source": "robots.txt", 
            "destination": "/robots.txt" 
        },
        { 
            "source": "sitemap.xml", 
            "destination": "/sitemap.xml" 
        },
        {
            "source": "/(.*)", 
            "destination": "/"
        }
    ]
}
```
* I need to access robots.txt file and sitemap.xml file via route.
* On localhost is not the problem because the files are being going normally.
*  When the route is outside the ones already defined it is redirected to a custom 404 page, that's what I want... but not with the sitemap.xml and robots.txt files

What could I do to resolve this?

**Comment by grinsteindavid** [comment]
it doesnt work for me either

**Reply by EuCarlos** [reply]
I put the _robot.txt_ and _sitemap.xml_ in the `public` directory as in the image and it worked for me
![image](https://user-images.githubusercontent.com/62575861/163462237-fd95c290-7f73-415b-8366-74cb47ad055a.png)
